dpc-admin/config/application.rb

```diff
@@ -15,6 +15,7 @@ class Application < Rails::Application
     config.load_defaults 6.0
 
     # Add fonts to asset pipeline
+    config.assets.prefix = '/admin_assets'
```

Comment:
I'm pretty sure we want this instead
```suggestion
    config.assets.prefix = '/admin/assets'
```

And then all of the `href="/admin_assets"` changes become `href="admin/assets"`.  If we _don't_ make this change, we have to configure our network systems to forward two paths to the admin app: `/admin` and `/admin_assets`.

Reply:
At first I tried serving the static assets from `/admin` but I couldn't get it to work. After some struggling I decided to add the following path patterns to the listener rule to accommodate:
https://github.com/CMSgov/dpc-ops/pull/328/files#diff-3b36f2c3ef0d4771518dfb7cb618a2754e8486290026465163af6008ca02ec24R157
Do you see any issues with this method going forward?

Reply:
@dhgreene fixed!